blog/2020-05-05-WorkflowProcessesWithAIScheduling.adoc

```diff
@@ -0,0 +1,579 @@
+= Workflow processes with AI scheduling
+:page-interpolate: true
+:awestruct-author: Christopher-Chianelli
+:awestruct-layout: blogPostBase
+:awestruct-tags: [useCase]
+
+A BPMN diagram models a Business Process. When the Business
+Process involves optimization at some step,
+https://www.optaplanner.org/[OptaPlanner] can help you quickly find an optimal solution.
+Now with https://kogito.kie.org/[Kogito], it is easier than ever before to integrate
+OptaPlanner into your BPMN diagrams! It so simple, that by the end of this blog,
+you will create a Business Process for scheduling a flight in an airline from scratch using
+OptaPlanner, BPMN and Quarkus in 15 minutes.
+
+== Project Setup
+
+Create a new folder to hold this project. Run the following command to
+create a new Quarkus project:
+
+[source]
+----
+mvn io.quarkus:quarkus-maven-plugin:1.4.1.Final:create \
+    -DprojectGroupId=com.company \
+    -DprojectArtifactId=airline-bpmn-optaplanner \
+    -Dextensions="resteasy, resteasy-jackson, optaplanner, \
+                  optaplanner-jackson, kogito-quarkus, smallrye-openapi"
+----
+
+== Creating your BPMN file
+
+We will be creating a simple BPMN that models the following process:
```

Comment:
Instead of "we will be creating", use "Create". Do this verb style consistently in the document for instructions.

General docs+article guidelines for consistent style:
- don't use "will <verb>", if you can just use "<verb>". So don't use future tense (the exception is likely the first blog paragraph where you state what the end result is at the end). For example "Ann will jump" -> "Ann jumps"
- avoid "are <verb>", just use "<verb>". For example "Ann is jumping" -> "Ann jumps"
- don't use "We <verb>", nor "I <verb>". Avoid "you <verb>", if you can just do "<verb>". For example: "I jump", "We jump", "You jump" -> "Jump"